meilenstein_03/uml-analyse-client/analyseclient.puml

```diff
@@ -7,8 +7,8 @@ package Analyse-Client{
     class Client {
         - data: Util.Data
         - dataForComp: Util.Data
-        - analyseTypes: List<AnalyseType>
-        - analyzedData: List<String>
+        - analyseTypes: AnalyseType[0..5]
```

Comment:
Muss doch eigentlich `AnalyseType[0..4]` sein da es nur 5 Elemente gibt in AnalyseType

Reply:
ne, damit ist die größe gemeint, nicht der index
int[2] bedeutet ja auch 2 elemente drin

Reply:
Ok ich dachte kurz damit wird die Index-Reichweite angegeben

Reply:
ja kann verwirrend sein, aber die größere Informationsfülle gleicht das finde ich aus